Classes/Service/Tika/ServerService.php

```diff
@@ -35,16 +32,29 @@
 /**
  * A Tika service implementation using the tika-server.jar
  *
+ * @copyright (c) 2015 Ingo Renner <ingo@typo3.org>
  */
 class ServerService extends AbstractService
 {
 
+    /**
+     * @var ClientInterface
+     */
+    protected $psr7Client;
+
+    /**
+     * List of valid status codes
+     *
+     * @var int[]
+     */
+    protected $validStatusCodes = [200, 202];
```

Comment:
Tika uses status code `204 - No content` if result is empty, we should consider this status as valid too

Reply:
Done.